core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerConfigService.java

```diff
@@ -157,13 +157,14 @@ public String getLoadBalancerUserFacingFQDN(Long stackId) {
         return loadBalancers;
     }
 
-    private boolean isLoadBalancerEnabled(StackType type, DetailedEnvironmentResponse environment) {
-        return isLoadBalancerEnabledForDatalake(type, environment) || isLoadBalancerEnabledForDatahub(type, environment);
+    private boolean isLoadBalancerEnabled(StackType type, String cloudPlatform, DetailedEnvironmentResponse environment) {
+        return isLoadBalancerEnabledForDatalake(type, cloudPlatform, environment) || isLoadBalancerEnabledForDatahub(type, environment);
     }
 
-    private boolean isLoadBalancerEnabledForDatalake(StackType type, DetailedEnvironmentResponse environment) {
+    private boolean isLoadBalancerEnabledForDatalake(StackType type, String cloudPlatform, DetailedEnvironmentResponse environment) {
         return StackType.DATALAKE.equals(type) && environment != null &&
-            (entitlementService.datalakeLoadBalancerEnabled(ThreadBasedUserCrnProvider.getAccountId()) ||
+            (cloudPlatform.equals("AWS") ||
```

Comment:
This check should be reversed to prevent an NPE, and use `CloudConstants`:
```
CloudConstants.AWS.equals(cloudPlatform)
```

Reply:
Ooh didn't think of that, will fix.

Reply:
Done